package.json

```diff
@@ -7,7 +7,7 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "start": "node app.js",
     "dev": "./node_modules/.bin/nodemon app.js",
-    "lint": "./node_modules/.bin/eslint 'app.js'"
+    "lint": "./node_modules/.bin/eslint './*/*.js'"
```

Comment:
So when it just says "app.js" the lint will only check that file before commit. when it has /*/*.js, it means check every .js file in the directory and their subfolders.